grammars/php.cson

```diff
@@ -721,9 +721,9 @@
         ]
       }
       {
-        'begin': '(?i)^\\s*(namespace)\\b\\s+(?=([a-z0-9_\\\\]+\\s*($|[;{]|(\\/[\\/*])))|$)'
+        'begin': '(?i)(^\\s*|\\s)(namespace)\\b\\s+(?=([a-z0-9_\\\\]+\\s*($|[;{]|(\\/[\\/*])))|$)'
```

Comment:
Shouldn't this be `\\s*` so that something like
```php
<?php          namespace superCoolNamespace
```
will work?

In addition, it looks like you can change that part to a non-capturing group, eg `(?:^\\s*|\\s*)` so that you don't have to change the capture group on L726.